labellab_mobile/lib/screen/project/edit_image/project_edit_image_screen.dart

```diff
@@ -3,6 +3,7 @@ import 'dart:io';
 import 'package:flutter/material.dart';
 import 'package:labellab_mobile/screen/project/edit_image/project_edit_image_bloc.dart';
 import 'package:labellab_mobile/screen/project/edit_image/project_edit_image_state.dart';
+import 'package:labellab_mobile/screen/project/edit_image/project_edit_image_state.dart';
```

Comment:
Is there any reason for duplicate imports?

Reply:
Must have automatically imported by the flutter plugin :slightly_smiling_face: